.github/workflows/ci_cd.yml

```diff
@@ -415,13 +322,18 @@ jobs:
               if 'npm' in eventConfig['distribute']:
                  npmPublishTag = eventConfig['distribute']['npm']
 
-          if isMainRepo and ((eventName == "release" and isRelease) or (eventName != "release" and not isRelease)):
+          if isMainRepo and (isRelease or (eventName != "release" and not isRelease)):
```

Comment:
This construct was there so it only tries to release on tags and not on the master branch. It could try to re-release for instance if you deploy a test instance just after creating a release. If there are no commits after just creating a release the release plugin resolves the version to a release and not a snapshot version.

Reply:
Thanks for the heads up.

The interaction with the versioning plugin and github actions seems a little confusing, I'll reinstate this for now.